src/fs_inode.hpp

```diff
@@ -37,18 +37,23 @@ namespace fs
                   const uint64_t  fusepath_len,
                   const mode_t    mode,
                   const dev_t     dev,
-                  const ino_t     ino);
+                  const ino_t     ino,
+                  const std::string &basepath = "");
```

Comment:
Is the default value really necessary? This would create a temp object if it was called with the default which isn't great.